xlive/H2MOD/GUI/imgui_integration/Console/CommandHandler.h

```diff
@@ -60,9 +67,18 @@ class ConsoleCommand
 		return m_max_parameter_count;
 	}
 
-    template<typename T = void*> T GetUserData() const 
-    { 
-        return (T)m_user_data; 
+    void SetCommandVarPtr(IComVar* varPtr)
+    {
+        if (varPtr != nullptr)
+        {
+            m_flags |= CommandFlag_SetsVariable;
```

Comment:
m_flags should use the c_flags class instead of duplicating logic

Reply:
replacing functionality with h2 code is not in the scope of this PR

Reply:
That's fine with me.